app/assets/javascripts/controllers/ansible_repository/repository_form_controller.js

```diff
@@ -0,0 +1,114 @@
+ManageIQ.angular.app.controller('repositoryFormController', ['$scope', 'repositoryId', 'miqService', 'API', function($scope, repositoryId,  miqService, API) {
+  var vm = this;
+
+  var init = function() {
+    vm.afterGet = false;
+
+    vm.repositoryModel = {
+      name: '',
+      description: '',
+      scm_type: 'git',
+      manager_resource: {},
+      scm_url: '',
+      scm_credentials: null,
```

Comment:
(Potentially a silly question as I not familiar with UI code)
There can 0 or 1 `scm_credential` associated with a `repo/project`. Here the pluralized form, does it mean multiple values expected? Or it is the list of values to be selected from?